src/DifferenceEquations.jl

```diff
@@ -1,6 +1,7 @@
 module DifferenceEquations
 
 using Distributions
+using DistributionsAD
```

Comment:
This might be needed or useful but generally it is a quite terrible dependency with heavy type piracy and usage of Requires.

Reply:
Thanks! Then which package should I use for auto differnetiability of the distributions?

Reply:
I guess you have to use DistributionsAD currently 🤷‍♂️ The plan is to fix the type piracies and to move parts to Distributions and other relevant packages. I don't have a better suggestion for now, I mainly want to point out the problems of DistributionsAD.

Reply:
Oh, I see. Then let's stick to this one until we've got better choices.